lib/index.js

```diff
@@ -82,44 +81,6 @@ module.exports = {
     if (overrides.hasBuildTimeWarning) {
       this.ui.writeWarnLine('[DEPRECATION] ' + overrides.buildTimeWarning);
     }
-
-    const { has } = require('@ember/edition-utils');
-
-    let optionalFeatures = this.project.addons.find((a) => a.name === '@ember/optional-features');
-    let optionalFeaturesMissing = optionalFeatures === undefined;
-
-    if (has('octane')) {
-      let message = [];
-
-      if (optionalFeaturesMissing) {
-        message.push(
-          `* the @ember/optional-features addon is missing, run \`ember install @ember/optional-features\` to install it`
-        );
-      }
-
-      if (message.length > 0) {
-        message.unshift(
-          `You have configured your application to indicate that it is using the 'octane' edition (via \`setEdition('octane')\`), but the appropriate Octane features were not enabled:\n`
-        );
-
-        throw new SilentError(message.join('\n\t'));
-      }
-    } else {
-      throw new SilentError(
-        'The Ember Classic edition has been removed. Specifying "classic" in your package.json, or not specifying a value at all, is no longer supported. You must explicitly set the "ember.edition" property to "octane". You can also run `npx @ember/octanify` to do this. \n\nFor more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_editions-classic'
-      );
-    }
-
-    if (
-      !optionalFeaturesMissing &&
-      optionalFeatures.isFeatureEnabled('jquery-integration') &&
-      typeof optionalFeatures.isFeatureExplicitlySet === 'function' &&
-      optionalFeatures.isFeatureExplicitlySet('jquery-integration')
-    ) {
-      throw new SilentError(
-        'Setting the `jquery-integration` optional feature flag to `true` was deprecated in Ember 3.x and removed in Ember 4.0.0. You must add the `@ember/optional-features` addon and set this feature to `false`.\n\nFor more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_optional-feature-jquery-integration'
-      );
```

Comment:
Not sure about this change. I _think_ we can simply remove these errors, but we could also keep them around (without the octane check).